tensorbay/client/gas.py

```diff
@@ -66,7 +66,7 @@ def _get_dataset_with_any_type(self, name: str) -> DatasetClientType:
         info = self._get_dataset(name)
         dataset_id = info["id"]
         is_fusion = info["type"]
-        commit_id = info["commitId"]
+        commit_id = info["commitId"] if info["commitId"] else ROOT_COMMIT_ID
```

Comment:
make OpenAPI to adapt this behavior,
before that, you can have a local patch for testing